packages/fluent/scss/forms/_layout.scss

```diff
@@ -0,0 +1,362 @@
+        .k-signature {
+            display: inline-flex;
+            width: 100%;
+        }
+    }
+
+    // Form Buttons Container
+    .k-form-buttons {
+        @extend .k-actions;
+        margin-top: calc( var( --kendo-form-field-margin-top, #{$kendo-form-field-margin-top} ) * 2 );
+        padding: 0;
+        overflow: visible;
+    }
+
+    // Form Hint/Error Messages
+    .k-form-hint,
+    .k-form-error {
+        margin-top: var( --kendo-form-hint-margin-top, #{$kendo-form-hint-margin-top} );
+        display: flex;
+        font-size: var( --kendo-form-hint-font-size, #{$kendo-form-hint-font-size} );
+        font-style: var( --kendo-form-hint-font-style, #{$kendo-form-hint-font-style} );
+    }
+
+    .k-text-start,
+    .k-buttons-start {
+        justify-content: flex-start;
+    }
+
+    .k-text-end,
+    .k-buttons-end {
+        justify-content: flex-end;
+    }
+
+    // Form Separator
+    .k-form-separator {
+        margin: var( --kendo-form-separator-margin, #{$kendo-form-separator-margin} );
+        border-width: 1px 0 0;
+        border-style: solid;
+        display: block;
+        flex: 0 0 auto;
+    }
+
+    // Horizontal Form
+    .k-form-horizontal {
+        .k-form-field {
+            display: flex;
+
+            > .k-label,
+            > kendo-label,
+            > .k-form-label {
+                margin-right: var( --kendo-horizontal-form-label-margin-x, #{$kendo-horizontal-form-label-margin-x} );
+                padding-top: var( --kendo-horizontal-form-label-padding-top, #{$kendo-horizontal-form-label-padding-top} );
+                width: var( --kendo-horizontal-form-label-width, #{$kendo-horizontal-form-label-width} );
+                text-align: end;
+                flex-direction: column;
+                align-items: var( --kendo-horizontal-form-label-align, #{$kendo-horizontal-form-label-align} );
+                justify-content: flex-start;
+            }
+            .k-label-optional {
+                margin: 0;
+                align-self: inherit;
+            }
+        }
+
+        .k-form-field-wrap {
+            max-width: var( --kendo-horizontal-form-field-wrap-max-width, #{$kendo-horizontal-form-field-wrap-max-width} );
+            flex: 1 1 auto;
+        }
+    }
+
+    // RTL Adjustments
+    .k-form {
+        [dir="rtl"],
+        .k-rtl & {
+            .k-label-optional {
+                margin-left: 0;
+                margin-right: var( --kendo-label-optional-margin-x, #{$kendo-label-optional-margin-x} );
+            }
+        }
+    }
+
+    .k-form-horizontal {
+        [dir="rtl"],
+        .k-rtl & {
+            &.k-form-field,
+            .k-form-field {
+                > .k-label,
+                > kendo-label,
+                > .k-form-label {
+                    margin-right: 0;
+                    margin-left: var( --kendo-horizontal-form-label-margin-x, #{$kendo-horizontal-form-label-margin-x} );
+                }
+            }
```

Comment:
check if these styles can be removed as well

Reply:
These styles are needed otherwise the horizontal form breaks in rtl